pkg/utils/config_checker.go

```diff
@@ -55,9 +56,14 @@ func NewConfigChecker(name string, configfiles ...string) (*configChecker, error
 		name:        name,
 		configfiles: configfiles,
 		checksum:    checksum,
+		reload:      true,
```

Comment:
I think you would want to set it as false by default

Reply:
Sure, we can do that.